src/components/profilePage/ProfileImage/ProfileImage.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./ProfileImagesStyles.css";
+import defaultPic from "../../../utils/icons/profile.png";
```

Comment:
Remove the profile.png file use icon font instead. This will lower one request and boost the performance.